layouts/partials/tw/nav.html

```diff
@@ -1,137 +1,311 @@
-<nav
+<div
   id="top"
   x-data="{ isOpen: false }"
```

Comment:
isOpen is obsolete now that it's replaced with toggle, so you can delete this line.

Reply:
This is removed: https://github.com/spotlightpa/poor-richard/pull/422/commits/cb0785ca062d3631356995b5fef37ca2b280ff9d